### PR TITLE
fix #37466 Activistpost.com

### LINKED ISSUE #37466: Activistpost.com
### Issue URL (Ads)
[http://Activistpost.com](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2FActivistpost.com)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | iOS
AdGuard version: | 3.0.1
Browser: | Safari
System wide filtering: | disabled
Simplified filters: | disabled
AdGuard DNS: | None
Filters: | AdGuard Mobile Ads,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media
Other extensions used: | None

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdguardFilters/issues/37466
ads script is masked as `.jpg` image

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -158,6 +158,7 @@ hdporncomics.com###sidebar-right > #text-3
 hdporncomics.com###sidebar-right > #custom_html-9
 ||analyticsindiamag.com/wp-content/uploads/2019/02/Blue_03.jpg
 analyticsindiamag.com##.textwidget > p > a[rel="nofollow external noopener noreferrer"] > img
+||activistpost.com/OeabcdRPf/VqFr/bcaeqn.jpg
```
Comment (AdamWr) [inline EnglishFilter/sections/specific.txt]: I'm afraid that the path/name of this script may be changed sometimes, so maybe something like this?
`||activistpost.com/*/*.jpg$script`

**Comment by LucasDiTomase** [comment]
as AdamWr pointed out, this rule works like a charm!

||activistpost.com//.jpg$script

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply EnglishFilter/sections/specific.txt]: agree with it

**Comment by tvinzz** [comment]
@AdamWr,
changed to `||activistpost.com/*/*.jpg$script`

**Review by AdamWr — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**